app/models/user.rb

```diff
@@ -51,7 +51,7 @@ def memberships_for(action, klass)
   def self.scope_for(action, user, opts={})
     case action
     when :show, :index
-      all
+      where(activated_state: 0)
```

Comment:
Can you try and use the activatable scopes here: https://github.com/zooniverse/Panoptes/blob/master/app/models/concerns/activatable.rb#L6

They need to change to:
``` ruby
scope :active, -> { where(activated_state: activated_states[:active]) }
scope :disabled, -> { where(activated_state: activated_states[:inactive]) }
```

Does this still get used in the controller actions? https://github.com/zooniverse/Panoptes/blob/master/app/controllers/api/v1/users_controller.rb#L34